lib/commands/element.js

```diff
@@ -185,12 +193,19 @@ const commands = {
     return {x: rect.x, y: rect.y};
   },
   /**
+   * Alias for {@linkcode XCUITestDriver.getLocation}
+   * @param {string|Element} elementId - the element ID
+   * @returns {Promise<Position>} The position of the element
+   * @deprecated Use {@linkcode XCUITestDriver.getElementRect} instead
    * @this {XCUITestDriver}
    */
-  async getLocationInView(el) {
-    return await this.getLocation(el);
+  async getLocationInView(elementId) {
+    return this.getLocation(elementId);
```

Comment:
typically we have followed the `return await` pattern as a style rule so it's clear just by looking (without needing a type checker) that we're dealing with an async function

Reply:
nothin gets by you, huh?? 😄

a few reasons why:

1. `await someFunction()` does not imply that `someFunction()` returns a `Promise`. Maybe `someFunction()` _used_ to return a `Promise`, and now you have potentially an entire chain of function calls which are `async` that needn't be.
2. if we always must use `return await someFunction()` then we are effectively documenting the return value of that function twice--and because of 1, this may be incorrect. this is akin to "no comments are better than wrong comments"
3. it is insignificantly slower (but it's slower)

if you want to pull rank, I'll change it, but I really hope to convince you & the team that this is unnecessary at best and harmful at worst.

Reply:
I also like to keep `await`. Readability is more important than a (very) minor performance gain

Reply:
if we were making this rule today i would agree with you given the extra layer of type safety that we now have to make clear what is a promise. at this point i'm purely concerned about inconsistency leading to confusion when reading code.

if you want to open another PR where you `s/return await/return/g` the entire codebase, i'm open to considering that, assuming other maintainers are OK with it as well.

Reply:
@mykola-mokhnach What of my first two points above?  It's not "good readability" if it's misleading.

Reply:
I would prefer to have an async function that is synchronous "internally" rather than async one that is never awaited properly. `await syncFunc()` is totally fine in JS and does not create any issues while `asyncFunc()` (e.g. `await` is missing) might  create serious problems in the code flow which are hard to track down.

Reply:
@mykola-mokhnach Given the state of our type support, we can be sure at this point that a missing `await` would fail the build.